.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
please remove this, it is not needed

Reply:
why is this file needed? We should avoid including this and `npm run lint` works fine without it :)